suites/reef/cephadm/tier-2-bootstrap-along-with-spec-file.yaml

```diff
@@ -0,0 +1,68 @@
+#============================================================
+# Test-Case: Bootstrap a cluster along with a spec file
+#
+# Cluster Configuration:
+#    conf/reef/cephadm/1admin-4node-1client-cluster.yaml
+#============================================================
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Bootstrap a cluster along with a spec file
+      desc: Bootstrap a cluster along with a spec file using "--apply-spec"
+      polarion-id: CEPH-83575604
+      module: test_cephadm_bootstrap_with_spec_file.py
+      config:
+        ansible_preflight: true
+        bootstrap:
+          mon-ip: node1
+          apply-spec: specs.yaml
+        client:
+          ansible_clients: true
+        specs:
```

Comment:
This specs should be under `bootstrap` key

Reply:
Thanks @vamahaja
Updated